frontend/server/src/Controllers/Problem.php

```diff
@@ -37,6 +37,7 @@
  * @psalm-type ProblemListPayload=array{currentTags: list<string>, loggedIn: bool, pagerItems: list<PageItem>, problems: list<ProblemListItem>, keyword: string, language: string, mode: string, column: string, languages: list<string>, columns: list<string>, modes: list<string>, tagData: list<array{name: null|string}>, tags: list<string>}
  * @psalm-type RunsDiff=array{guid: string, new_score: float|null, new_status: null|string, new_verdict: null|string, old_score: float|null, old_status: null|string, old_verdict: null|string, problemset_id: int|null, username: string}
  * @psalm-type CommitRunsDiff=array<string, list<RunsDiff>>
+ * @psalm-type ProblemCollectionDetails=array{collection: list<string>}
```

Comment:
De acuerdo a los cambios propuestos, esto podría quedar así:
```suggestion
 * @psalm-type ProblemCollectionDetails=array{collection: list<array{alias: string, name?: string}>}
```